pkg/discoveredpolicy/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Can we change this to something like this ? @nyrahul wdyt ?
```suggestion
/*
 * Accuknox Proprietary Software License
 * (c) 2023 Accuknox. All rights reserved.
 * Unauthorized use, copying, or distribution of this software is strictly prohibited.
 * This software is provided "as is" without warranties or support of any kind.
 * For inquiries, please contact Accuknox at info@accuknox.com.
 */
```

Reply:
@nyrahul any thoughts here?

Reply:
No point in keeping any license at all.

Reply:
removed license part across all the files.